app/views/wiki/dictionary/_item.html.erb

```diff
@@ -0,0 +1,3 @@
+<div>
+  <%= link_to item[1]["en-US"], wiki_article_path(CGI.escape(item[1]["en-US"]).downcase) %>
```

Comment:
Is there *any* possible chance we can make this use the `I18n.current_locale` somehow?

Reply:
That is a good idea, however I just checked and we don't have Korean translations for any of the values or actions, so we'd still only have English.